swift/apple/FirezoneKit/Sources/FirezoneKit/Models/Configuration.swift

```diff
@@ -25,7 +25,13 @@ public struct Configuration: Codable {
   var firezoneId: String?
   var internetResourceEnabled: Bool?
 
-  public init(from dict: [String: Any?]) {
+  // Keys from managed configuration we want to mask the app configuration with if set.
+  // This will prevent the user from modifying them in the app.
+  private let overriddenKeys: [String]
+
+  public init(from dict: [String: Any?], overriddenKeys: [String] = []) {
+    self.overriddenKeys = overriddenKeys
```

Comment:
Should this be computed based on two dicts that you pass in the constructor?

Reply:
I ended up refactoring this quite a bit to do this and other nicer things.